### PR TITLE
RSP3a3

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: Spec/RestClientPresence.swift [hunk 1]
```diff
@@ -75,6 +75,48 @@ class RestClientPresence: QuickSpec {
...
+                    }
+
+                    waitUntil(timeout: testTimeout) { done in
+                        disposable += AblyTests.addMembersSequentiallyToChannel("test", members: 3, startFrom: 7, options: options) {
+                            done()
+                        }
+                    }
+
+                    let query = ARTRealtimePresenceQuery()
+                    query.connectionId = disposable.last!.connection.id!
```
Comment (mattheworiordan) [inline Spec/RestClientPresence.swift]: Mind adding a comment or explaining why the connection ID of the last disposable only returns the last 3?

Comment (ricardopereira) [inline Spec/RestClientPresence.swift]: Done aed0a15.

File: Source/ARTPresence+Private.h [hunk 2]
```diff
@@ -11,6 +11,12 @@
 
 #import "ARTPresence.h"
 
+@interface ARTPresenceQuery ()
+
+- (NSMutableArray /* <NSURLQueryItem *> */ *)asQueryItems;
```
Comment (ricardopereira) [inline Source/ARTPresence+Private.h]: Why not `- (__GENERIC(NSMutableArray, NSURLQueryItem *) *)asQueryItems;`?

**Comment by ricardopereira** [comment]
Small comment but LGTM.

File: Source/ARTPresence+Private.h (hunk 2 above)
Comment (tcard) [inline Source/ARTPresence+Private.h]: Fixed at 0aebf60.

**Pull request merged**